concent_api/concent_api/tests/test_check_signing_service_public_key.py

```diff
@@ -0,0 +1,60 @@
+from django.test import override_settings
+from django.conf import settings
+from django.test import TestCase
+
+from common.testing_helpers import generate_ecc_key_pair
+from concent_api.system_check import check_signing_service_public_key
+from concent_api.system_check import create_error_48_signing_service_public_key_is_not_set
+from concent_api.system_check import create_error_49_signing_service_public_key_is_invalid
+
+
+(SIGNING_SERVICE_PRIVATE_KEY, SIGNING_SERVICE_PUBLIC_KEY) = generate_ecc_key_pair()
+
+
+class TestSigningServicePublicKeyCheck(TestCase):
+
+    @override_settings(
+        CONCENT_FEATURES=[
+            'middleman'
+        ]
+    )
+    def test_that_signing_service_public_key_not_set_will_produce_error_when_middleman_is_in_concent_features(self):
+        del settings.SIGNING_SERVICE_PUBLIC_KEY
+
+        errors = check_signing_service_public_key()
+
+        self.assertEqual(len(errors), 1)
+        self.assertEqual(errors[0], create_error_48_signing_service_public_key_is_not_set())
+
+    @override_settings(
+        CONCENT_FEATURES=[]
+    )
```

Comment:
Same situation as previous. Which style do we want to use, because breaking one line statement might be overkilling

Reply:
As I have said, I think it is good for readability. It doesn't have any impact on performance or size of result code in `pyc` files.